core/src/test/resources/test_assets.yaml

```diff
@@ -157,24 +151,11 @@ assets:
     sep31:
       enabled: true
       receive:
-        fee_fixed: 0
-        fee_percent: 0
         min_amount: 1
         max_amount: 1000000
       quotes_supported: true
       quotes_required: true
       # todo assume this is right for now
-      fields:
-        transaction:
-          receiver_routing_number:
-            description: routing number of the destination bank account
-          receiver_account_number:
-            description: bank account number of the destination
-          type:
-            description: type of deposit to make
-            choices:
-              - SEPA
```

Comment:
How does the wallet specify the `type` now? SEP-6 defines a `method` configuration, do we plan to add this for SEP-31 in another PR?

Reply:
The `fields` is optional and deprecated in [SEP-31](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#asset-object-schema). 

How do we use the fields for SEP-6?

Reply:
Per our discussion, I will create another PR to add sep31.receive.methods
[ANCHOR-809](https://stellarorg.atlassian.net/browse/ANCHOR-809)